### PR TITLE
CFE-2550 Change: Suppress undesired errors

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: inventory/any.cf [hunk 1]
```diff
@@ -787,8 +787,13 @@ bundle agent inventory_cmdb_load(file)
       "$(ckeys)" expression => "any", scope => "namespace";
 
   vars:
-      "cmdb" data => readjson($(file), "999999");
-      "cmdb_string" string => format("%S", cmdb);
+      "cmdb"
+        data => readjson($(file), "999999"),
```
Comment (tzz) [inline inventory/any.cf]: Does the number need to be quoted?

**Comment by tzz** [comment]
YES for me

File: inventory/any.cf (hunk 1 above)
Comment (nickanderson) [inline inventory/any.cf]: Probably not, but it was already quoted so i left it.

**Comment by nickanderson** [comment]
Syntax error .... double parenthesis slipped in, pushed fix

**Pull request merged**